### ISSUE TITLE
Feature-Wunsch: Auslöser der 404 Seite erfassen

### ISSUE THREAD

**Issue by BugBuster1701** [BugBuster1701/contao-visitors-bundle#33]
[Forum](https://community.contao.org/de/showthread.php?70669-Viele-Error404-Zugriffe-in-der-Visitors-Statistik-sichtbar&p=472915&viewfull=1#post472915)

**Comment by Schmidty2** [comment]
Ein super Feature wäre es dann noch, wenn man die 404er unterscheiden könnte in Crawler und tatsächliche Seitanaufrufe, die einen 404er erzeugt haben. Und wenn man dann die Crawler noch ausblenden könnte wäre das super.

**Comment by Ernestopheles** [comment]
Ja, das finde ich auch. Wichtig wäre ja zu wissen, welche Seite versucht wurde aufzurufen.

**Comment by BugBuster1701** [comment]
Kurzer Hinweis: Wenn ein Crawler den Request ausgeführt hat und dieser von Visitors über Botdetection erkannt wurde, dann sollte der 404 Aufruf gar nicht gezählt werden (werde ich nochmals prüfen)
Dazu gibt es die Erweiterung BotStatistik. Ob die auch die 404er mit beachtet kann ich grad nicht sagen, werde mir das mal anschauen.

**Comment by BugBuster1701** [comment]
Notiz für mich: Erkennung über `$objPage->type = 'error_404'` möglich.

**Comment by Anke** [comment]
+1
